puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_cli.yaml

```diff
@@ -28,17 +28,10 @@
     builders:
```

Comment:
Do we want to drop this file already or do you keep them in parallel so we can monitor it for a bit?